### DISCUSSION TITLE
`origin.producer` when there are multiple producers

### DISCUSSION THREAD

**Discussion by lucasrodes** [owid/etl#1608] [Metadata] [answered]
The [Human Mortality Database](https://www.mortality.org/Data/ZippedDataFiles) has multiple producers. In their explanatory notes, the following full citation is shown:

> HMD. Human Mortality Database. Max Planck Institute for Demographic Research (Germany), University of California, Berkeley (USA), and French Institute for Demographic Studies (France). Available at [www.mortality.org](http://www.mortality.org/).

From this, I understand that there are three producers of the dataset:
- Max Planck Institute for Demographic Research (Germany)
- University of California, Berkeley (USA)
- French Institute for Demographic Studies

What should be the value for `origin.producer`?

I was thinking of concatenating all the producers with a `,`, i.e.:

> Max Planck Institute for Demographic Research, University of California, French Institute for Demographic Studies

**Comment by pabloarosado** [comment, marked as answer]
In this case I'd suggest the producer is "Various sources", and the data product "The Human Mortality Database". Then, in `citation_full` you disclose all sources in detail.

**Reply by lucasrodes** [reply]
Thanks, @pabloarosado! 

Do you think there is anything to consider for the field `attribution` in such cases? E.g., do you think the automatic format "Various sources (2023)" is fine?

**Reply by pabloarosado** [reply]
I think the automatic format should be fine. I've used it for example for the dataset on bullfighting laws ([this chart](https://owid.cloud/admin/charts/7123/edit)), which is a compilation of many sources. We currently don't have any guidelines on when exactly a group of sources should become "Various sources", but in your case I think it's a fair choice.
